ros__parameters:
  SOURCE: /zed2/zed_node/rgb/image_rect_color
  DEPTH_INPUT: /zed2/zed_node/depth/depth_registered
  CAMERA_INFO: /zed2/zed_node/depth/camera_info
  ROS_INPUT: True
  USE_ACTIVE_FLAG: False
  VERBOSE: False
  DEPTH_ACTIVE: True
  USE_YOLO8: False
  YOLO_MODEL_PATH: /path/to/models/yolo11classes.pt
  MODELS_PATH: /path/to/models/gazebo/
  LABELS_PATH: /path/to/models/gazebo/label_map.pbtxt
  CAMERA_FRAME: zed2_left_camera_optical_frame
